dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/types/validation-error-generator.ts

```diff
@@ -3,6 +3,8 @@ import { UtilisationReportDataValidationError, UtilisationReportCsvCellData, Uti
 export type UtilisationReportCellValidationErrorGenerator = (
   cellData: UtilisationReportCsvCellData | undefined,
   exporterName?: string | null,
-) => UtilisationReportDataValidationError | null;
+) => UtilisationReportDataValidationError | null | Promise<UtilisationReportDataValidationError | null>;
 
-export type UtilisationReportRowValidationErrorGenerator = (rowData: UtilisationReportCsvRowData) => UtilisationReportDataValidationError | null;
+export type UtilisationReportRowValidationErrorGenerator = (
+  rowData: UtilisationReportCsvRowData,
+) => UtilisationReportDataValidationError | null | Promise<UtilisationReportDataValidationError | null>;
```

Comment:
Shouldn't these always have a promise?

```ts
Promise<UtilisationReportDataValidationError | null>
```

Instead of 

```
UtilisationReportDataValidationError | null | Promise<UtilisationReportDataValidationError | null>
```

Reply:
Interesting, struggling to find examples of them being async and not async, bit confusing but fully trust you 👍 

FYI @Zainzzkk